src/config/src/meta/dashboards/mod.rs

```diff
@@ -19,16 +19,6 @@ use utoipa::ToSchema;
 
 type OrdF64 = ordered_float::OrderedFloat<f64>;
 
-#[derive(Clone, Debug, Serialize, Deserialize, ToSchema)]
-pub struct DashboardVersion {
-    #[serde(default = "default_version")]
-    pub version: i32,
-}
-
-fn default_version() -> i32 {
-    1
-}
-
```

Comment:
This is only used in deserialization from REST HTTP requests so it is moved into the `/http` module